frontend/src/components/header.js

```diff
@@ -133,16 +133,14 @@ const Header = () => {
               {locationsList.map((item, index) => (
                 <Link
                   key={`browseLink-${index}`}
-                  to={item.route}
-                  target="_blank"
+                  href={item.route}
```

Comment:
for location list, need to use `to` instead 'href' you can do ternary opration for the which Link and properties to use:
Here is my suggestion 
```
index === 0 ? (<Link key={`browseLink-${index}`} to={item.route}  className={`${textStyle} hover:bg-[#0C2966] py-2 px-4 border-b-[0.1px] border-gray-100 tracking-[0.96px]`}>{item.title}</Link>) : (<Link key={`browseLink-${index}`} href={item.route}  className={`${textStyle} hover:bg-[#0C2966] py-2 px-4 border-b-[0.1px] border-gray-100 tracking-[0.96px]`}>{item.title}</Link>)
```
i think this works.  i do not know full syntax for reactjs for this ternary operation.  but hopefully it works.

Reply:
ok ive fixed this